detection_rules/cli_utils.py

```diff
@@ -208,19 +236,19 @@ def rule_prompt(path=None, rule_type=None, required_only=True, save=True, verbos
         else:
             if skip_errors:
                 # return missing information
-                return f"Rule: {kwargs["id"]}, Rule Name: {rule_name} is missing {name} information"
+                return f"Rule: {kwargs['id']}, Rule Name: {rule_name} is missing {name} information"
             else:
                 result = schema_prompt(name, is_required=name in required_fields, **options.copy())
         if result:
-            if name not in required_fields and result == options.get('default', ''):
+            if name not in required_fields and result == options.get("default", ""):
                 skipped.append(name)
                 continue
 
             contents[name] = result
 
     # DEFAULT_PREBUILT_RULES_DIRS[0] is a required directory just as a suggestion
-    suggested_path = Path(DEFAULT_PREBUILT_RULES_DIRS[0]) / contents['name']
-    path = Path(path or input(f'File path for rule [{suggested_path}]: ') or suggested_path).resolve()
+    suggested_path: Path = Path(DEFAULT_PREBUILT_RULES_DIRS[0]) / contents["name"]
+    path = Path(path or input(f"File path for rule [{suggested_path}]: ") or suggested_path).resolve()
```

Comment:
Seems a bit odd to type hint as Path when we explicitly set as a Path object. We also dont type hint the next field `path`.

Reply:
`contents` is `dict[str, Any]`, so there is ambiguity in the calculation of the result type